frontend/src/modules/integration/integration-service.js

```diff
@@ -348,4 +348,48 @@ export class IntegrationService {
 
     return response.data.isWebhooksReceived;
   }
+
+  static async groupsioConnect(email, token, groupNames) {
+    const tenantId = AuthCurrentTenant.get();
```

Comment:
When you add this to crowd-linux, don't forget to add segments to these requests 😄